docs/concepts/artifact.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: OCI artifact
+sidebar_position: 1
+---
+
+# Understanding OCI artifacts
+
+The Open Container Initiative (OCI) organization has played
+a crucial role in defining the industry-standard specifications for container
+formats and runtime and artifacts.
+OCI artifacts encompass an extensive variety of content types, from signatures,
+Software Bill of Materials (SBOMs), Helm charts, container images, to Open Policy Agent (OPA)
+bundles and more, thereby lending incredible flexibility to how containerized
+applications are packaged, delivered, and updated.
+
+This article will delve into OCI artifacts created using a manifest defined as a part of
+the OCI image specification.
+
+## The Anatomy of an OCI artifact
+
+The structure of an OCI Artifact includes an Image Manifest or an Image Index
+that points to other OCI artifacts. These artifacts can be stored and accessed from
+[content addressable storage][cas] like a registry, or an on-disk storage like
+an [OCI-Layout][oci-layout] directory.
+
+The OCI artifact can be referenced usig a *tag*
+or a *digest*. The digest is a hash of the OCI artifact manifest or index and
+should be assumed to be immutable where as the tag may be mutable.
+
+A *tag* resolves to a data structure called the [descriptor][descriptor] which
+contains the digest of the manifest or index.
+
+![Diagram showing relationship and fiels of a tag image and index](/img/concepts/artifact/tag_relations.png)
+
+### The Significance of Annotations
+
+Annotations in OCI artifacts offer a means of adding metadata to various
+components, including the Image Manifest, Image Index and Image Layers.
+These annotations hold key-value pairs that represent either the metadata
+of the OCI artifact or blobs, such as the creation time, or data of the artifact itself.
+
+```json
+{
+  ...
+  "annotations": {
+    "oci.opencontainers.image.created": "2023-01-02T03:04:05Z",
+    "com.example.information": "useful-info"
+  }
+}
+```
+
+### Determining the artifact type
+
+For OCI artifacts the **type** of the artifact may be determined from the `artifactType` property
+in the image manifest or index. This property was introduced in v1.1 of the [image specification][image-properties].
```

Comment:
also, suggest keeping Index here, to foster creative ideas, and ack future support potential.